### ISSUE TITLE
Removal of Liveness Probe in Loki 3.x for Query Frontend

### ISSUE THREAD

**Issue by kashish1200** [grafana/loki#13967]
I’ve been reviewing the Loki 3.x configuration, and I noticed that the liveness probe is no longer present for certain components, specifically the Query Frontend and Gateway. In previous versions, these components did have liveness probes configured.

Could you please provide some insight into why the liveness probes were removed for these components in Loki 3.x?

**Comment by chaudum** [comment]
@kashish1200 Is this question regarding the Helm chart?

**Comment by kashish1200** [comment]
@chaudum Yes, my question is related to the Helm chart.

**Comment by chaudum** [comment]
> Could you please provide some insight into why the liveness probes were removed for these components in Loki 3.x?

I don't think that there were particular reasons for removing them, they probably got forgotten about in the rewrite of the Helm chart for 3.0

**Comment by JStickler** [comment]
As of March 16, 2026, the Loki Helm Chart has been forked to the new [Grafana-Community/helm-charts repo](https://github.com/grafana-community/helm-charts) and is now being maintained by the Grafana Community and Grafana Champions. If this issue affects Grafana Enterprise Logs (GEL), please leave a comment here.  Otherwise, if this is still an issue, please reopen your issue here [here](https://github.com/grafana-community/helm-charts/issues).
